test/pylib/rest_client.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""Asynchronous helper for Scylla REST API operations.
+"""
+import os.path
+import aiohttp
+
+
+class UnixRESTClient():
+    """An async helper for REST API operations using AF_UNIX socket"""
+    session: aiohttp.ClientSession
```

Comment:
Is this field declaration necessary for anything? I see that `sock_name` is not declared, for example.

Reply:
Removed in https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d